frontend/src/layout/navigation-3000/sidepanel/panels/activation/activationLogic.ts

```diff
@@ -25,6 +24,7 @@ export enum ActivationTasks {
     SetupSessionRecordings = 'setup_session_recordings',
     TrackCustomEvents = 'track_custom_events',
     InstallFirstApp = 'install_first_app',
+    SetupReverseProxy = 'setup_reverse_proxy',
```

Comment:
nit: `setup` is a noun, `set up` is the verb. 
```suggestion
    SetUpReverseProxy = 'set_up_reverse_proxy',
```